react-facet/docs/api/hooks.md

```diff
@@ -38,9 +37,11 @@ interface Props {
 const Form = ({ onSubmit, initialValue }: Props) => {
   const [value, setValue] = useFacetState(initialValue)
 
-  const handleChange = useCallback(
+  const handleChange = useCallback<KeyboardCallback>(
     (event) => {
-      setValue((event.target as HTMLInputElement).value)
+      if (event.target instanceof HTMLInputElement) {
```

Comment:
It'd be nice to not have to do this check. 
AFAIK, it's either this or type casting `target` to `HTMLInputElement`. 
Could `KeyboardCallback` be more strict? Something like `KeyboardCallback<HTMLInputElement>`